### ISSUE TITLE
Adding old lab-caches (during Mega-Events)

### ISSUE THREAD

**Issue by SD1313** [mirsch/lab2gpx#15]
I try to get also older lab-caches as gpx, but they maybe hidden from the actual api-call.
It would be nice, if there would be an possibility also add known id's manualy.
As example https://labs.geocaching.com/Adventures/Details/53fad7db-a470-43d8-b5bc-b7bc0240ba74 from Erfurt.

**Comment by mirsch** [comment]
How do you know the id if it's hidden in the API?

**Comment by SD1313** [comment]
I made no research yet. You can get the id if you logged the cache, when you take a look to the Log's-Page and copy the link from the header-image of the lab. So for me it's interesting, to get also my found's as gpx-File, that i found in the past (before adventure-labs getting popular)

**Comment by wabicek** [comment]
yeah, it would be cool to be able download all found lab caches as gpx.

**Comment by SD1313** [comment]
I think this request is the same as in #56.
The function should be the oposite of #44

**Comment by SamHenkel** [comment]
> 
> 
> @mirsch
> I think this request is the same as in #56.
No it's just similar. My intention is to select AdventureLabs with the help of the regulare GeocacheApi, mapping the result and then select AdLabs geographically in order to a planned route. I'd like then loading Labcaches with a resulting guid-string with this tool.
Hans

**Comment by Billy-Wallace** [comment]
A call can be made to the API for all Adventure Labs from a given location to get all the GUIDs.
`adventures/search?q=location:[$location]&skip=0&take=0&fields=id&excludeCompleted=true`

Then those GUIDs can be checked against all the Adventure Labs already stored locally in a database.

This would leave you with a list of missing GUIDs

The ability to paste this list of missing GUIDs into an Include GUIDs text area would be really valuable, being very small, kind to bandwidth, kind to GS servers loads and makes good sense all around.

**Comment by olpp** [comment]
Hi, I would like to know, why some older labs are not included in the response and if there might be a general solution?
I have the same problem with "Movies Movies Movies" available only a few days starting on the Gigaevent "GIFF 2018 - Janz jroß Kinema" (GC7QR3N).
Another lab "Discover Colonia" released for this Gigaevent was available much longer and is shown in the gpx requested by Lab2Gpx.
What is the reason for the "Movies Movies Movies" missing and can I solve the problemsomehow?
Thanks Ulf

**Comment by olpp** [comment]
Actually I still don't understand how to get those old labs.
